### PR TITLE
Update api versions

### PR DESCRIPTION
**Pull request by FranBarrera**

### CODE REVIEW

**Review by antoineco — APPROVED** [review]

File: bridges/sqs-twilio/sqs-twilio.yaml [hunk 1]
```diff
@@ -7,13 +7,13 @@ metadata:
 spec:
   components:
     - object:
-        apiVersion: eventing.knative.dev/v1beta1
+        apiVersion: eventing.knative.dev/v1b
```
Comment (odacremolbap) [inline bridges/sqs-twilio/sqs-twilio.yaml:10]: typo!

File: bridges/cronjob-tekton/cron-tekton.yaml [hunk 2]
```diff
@@ -7,22 +7,23 @@ metadata:
...
-        apiVersion: sources.knative.dev/v1alpha2
+        apiVersion: sources.knative.dev/v1
         kind: PingSource
         metadata:
           name: minute-cron-job-source-tm
         spec:
           schedule: "* * * * *"
-          jsonData: '{"buildtype": "task", "name": "tekton-test", "params":{"greeting": "Hi from Triggermesh EveryBridge!"}}'
+          contentType: "application/json"
+          data: '{"buildtype": "task", "name": "tekton-test", "params":{"greeting": "Hi from Triggermesh EveryBridge!"}}'
```
Comment (cab105) [inline bridges/cronjob-tekton/cron-tekton.yaml:23]: I'm kinda curious, why the switch from `jsonData` to `data` and `contentType`?

**Review by odacremolbap — CHANGES_REQUESTED** [review]
thanks.

I spotted a small typo.

File: bridges/sqs-twilio/sqs-twilio.yaml (hunk 1 above)
Comment (cab105) [inline bridges/sqs-twilio/sqs-twilio.yaml:10]: Typo

**Review by cab105 — CHANGES_REQUESTED** [review]
Left a small question, and found a typo that would break a bridge.

**Comment by cab105** [comment]
Don't forget, the bridge files that do find their way in the catalog are located at: https://github.com/triggermesh/config/tree/main/base/bridges/catalog

File: bridges/sqs-twilio/sqs-twilio.yaml (hunk 1 above)
Comment (FranBarrera) [reply bridges/sqs-twilio/sqs-twilio.yaml:10]: Fixed https://github.com/triggermesh/bridges/pull/91

Comment (FranBarrera) [reply bridges/sqs-twilio/sqs-twilio.yaml:10]: Fixed https://github.com/triggermesh/bridges/pull/91

File: bridges/cronjob-tekton/cron-tekton.yaml (hunk 2 above)
Comment (FranBarrera) [reply bridges/cronjob-tekton/cron-tekton.yaml:23]: Following the latest documentation jsonData is replaced with data and added contentType. https://knative.dev/docs/developer/eventing/sources/ping-source

**Pull request merged**